client/flutter/lib/pages/home_page.dart

```diff
@@ -68,8 +80,11 @@ class _HomePageState extends State<HomePage> {
                 PageButton(
                   Color(0xff008DC9),
                   S.of(context).homePagePageButtonProtectYourself,
-                  () => Navigator.of(context).push(
-                      MaterialPageRoute(builder: (c) => ProtectYourself())),
+                  () {
+                    _logAnalyticsEvent('ProtectYourself');
```

Comment:
We'll want clearer event info in the future. `ProtectYourself` could mean a number of things, we should specify that this is a button press and/or a view of a screen

Reply:
We don't even need to track page routing manually, the navigation observer will do this for us automatically - but currently our routing process does not have routeSettings.name, so it won't work. Check for detailed info : https://github.com/WorldHealthOrganization/app/pull/512#discussion_r399769421
This was done temporarily for v1, but a PR with better routing is waiting https://github.com/WorldHealthOrganization/app/pull/538

Reply:
not a blocker for now, can update later